test/Directory.Build.props

```diff
@@ -2,7 +2,7 @@
   <Import Project="..\Directory.Build.props" />
 
   <PropertyGroup>
-    <DeveloperBuildTestTfms>net9.0</DeveloperBuildTestTfms>
+    <DeveloperBuildTestTfms>$(NetCurrent)</DeveloperBuildTestTfms>
```

Comment:
@deepchoudhery Do you know why we have both of these variables? `DeveloperBuildTestTfms` and `StandardTestTfms`? It looks like the former is only used to set the latter based on a quick search of the repo, but I might have missed something. Seems like it'd be easier to just set `<TargetFramework>$(NetCurrent)</TargetFramework>` here in D.B.props and remove it from all the test projects. But I didn't want to do all that in this PR - was just trying to unbreak the build.

Reply:
ye no need for both of these vars, we can just use $(NetCurrent) in the test .csproj files. Did not know that $(NetCurrent) was a thing.

Reply:
I can do it after we merge this.